state-chain/pallets/cf-elections/src/benchmarking.rs

```diff
@@ -201,7 +201,7 @@ mod benchmarks {
 
 		#[block]
 		{
-			let _ = Pallet::<T, I>::recheck_contributed_to_consensuses(epoch, &validator_id, epoch);
+			let _ = Pallet::<T, I>::recheck_contributed_to_consensuses(epoch, &validator_id, 0);
```

Comment:
What exactly was the problem here? I would think that epoch should be zero by default (at least judging by the looks of `impl_mock_epoch_info`)?